### PR TITLE
m1 bindings and members scattered around

### PR DESCRIPTION
**Pull request by MarioMastr**
this is a big one so i'll be perfectly fine with changing anything that looks wrong that i may have mised

### CODE REVIEW

**Comment by altalk23** [comment]
what are the 0x0 pads they are useless girl

File: test/members/MacOSArm.cpp [hunk 1]
```diff
@@ -30,6 +30,34 @@ GEODE_MEMBER_CHECK(GJBaseGameLayer, m_sections, 0x32f8);
 GEODE_MEMBER_CHECK(GJBaseGameLayer, m_audioVisualizerBG, 0x3420);
 GEODE_MEMBER_CHECK(GJBaseGameLayer, m_loadingLayer, 0x3498);
 
+GEODE_MEMBER_CHECK(EnterEffectInstance, m_gameObject, 0x18);
+GEODE_MEMBER_CHECK(EnterEffectInstance, m_unkVecInt, 0x20);
+
+GEODE_MEMBER_CHECK(CountTriggerAction, m_unkVecInt, 0x0);
```
Comment (altalk23) [inline test/members/MacOSArm.cpp:36]: what in the world is this blasphemy

**Comment by MarioMastr** [comment]
wait a second those commits were added to the pr? LOL mb i didnt think they would be

File: bindings/2.206/Extras.bro [hunk 2]
```diff
@@ -8,20 +8,20 @@ class AdvancedFollowInstance {
...
 
 class SFXTriggerInstance {
-	PAD = win 0x10;
+	PAD = win 0x10, m1 0x0;
 	SFXTriggerGameObject* m_sfxTriggerGameObject;
 }
 
 class EventTriggerInstance {
-	PAD = win 0x10;
+	PAD = win 0x10, m1 0x0;
```
Comment (altalk23) [inline bindings/2.206/Extras.bro:16]: yeah please do not use pad 0x0 anywhere that just defeats the purpose and it doesnt even work

**Review by altalk23 — CHANGES_REQUESTED** [review]

File: bindings/2.206/Extras.bro (hunk 2 above)
Comment (MarioMastr) [reply bindings/2.206/Extras.bro:16]: the thing is if i dont put them there the members are privated

Comment (altalk23) [reply bindings/2.206/Extras.bro:16]: then put in the correct pads

Comment (MarioMastr) [reply bindings/2.206/Extras.bro:16]: that's the problem, i can't see any

**Comment by MarioMastr** [comment]
you know what i think im just going to close this one for now

**Pull request closed without merging**